app/components/Reactions/Reaction.js

```diff
@@ -34,10 +35,11 @@ class Reaction extends React.Component<Props> {
       addReaction,
       deleteReaction,
       hasReacted,
+      canReact,
       reactionId,
       contentTarget,
     }: Props = this.props;
-    const classes = [className ? className : styles.reaction];
+    const classes = [className ? className : styles.reaction + (canReact ? " "+styles.clickable : "")];
```

Comment:
We use `classNames` here, hence the array. So it's better to add the classname as an element in the array.

A suggestion is to do something like this. (or something cleaner if you can :smile:)
```js
const classes = [className ? className : styles.reaction, canReact && styles.clickable].filter(Boolean) 
```